steps/mentor-calibre-lvs/configure.yml

```diff
@@ -28,6 +28,7 @@ outputs:
 commands:
   - envsubst < lvs.runset.template > lvs.runset
   - if [ ! -f inputs/*.svrf ]; then touch inputs/rules.svrf; fi
+  - cat inputs/*.lvs.v > inputs/merged.v
```

Comment:
Can you cat this to the current directory instead of into inputs? Usually we want inputs to only be whatever came into the step. You shouldn't use it as a working directory.